.github/workflows/ios.yml

```diff
@@ -0,0 +1,35 @@
+name: iOS
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'src/**'
+      - 'examples/**'
+      - '.github/workflows/android.yml'
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'examples/**'
+      - '.github/workflows/android.yml'
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Build Xcode15 project
+      run: |
+        cd projects/Xcode15
+        curl -L https://github.com/raysan5/raylib/files/14743869/libEGL.xcframework.zip --output libEGL.xcframework.zip

```

Comment:
Where are those files located? It looks a bit dodgy... isn't there a better approach?